shark_turbine/kernel/ops/wave_ops.py

```diff
@@ -0,0 +1,238 @@
+from abc import ABC
+from dataclasses import dataclass, field
+from functools import wraps
+import sys
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Optional,
+    Sequence,
+    Type,
+    TypeVar,
+    TypedDict,
+    final,
+)
+import torch.fx as fx
+
+if TYPE_CHECKING:
+    from ..lang.functional_types import Memory, Register
+from .._support.indexing import IndexExpr
+from .._support.dtype import DataType
+from .base import OpDispatcher
+
+
+def get_node_name(string: str, skip_first: bool = True):
```

Comment:
The fx node name depends on previous nodes in the graph, e.g. it might be `read_6`. This function extracted a readable name from our node classes. I removed it as we already give ops a name in the new `define_op` [here](https://github.com/iree-org/iree-turbine/blob/da97c1c9de86daa557902f0b0560a2f49301aa85/shark_turbine/kernel/ops/wave_ops.py#L214) and can simply use that instead.